cristin-commons/src/main/java/no/unit/nva/cristin/model/SearchResponse.java

```diff
@@ -127,7 +127,7 @@ public void setPreviousResults(URI previousResults) {
         this.previousResults = previousResults;
     }
 
-    public List<?> getHits() {
+    public List<E> getHits() {
```

Comment:
why are you doing this? it seems to work but just wondering what is the difference between `E `and `? ` and further down you still use `?`

Reply:
this has a big impact as a lot of queries use this class. but if the karate tests pass it should not be a problem

Reply:
forget everything i said here. it seems the `?` were a remnant of the old pattern as i can see all other places in the model it now uses `E`